de_web_plugin.cpp

```diff
@@ -4987,6 +5163,12 @@ void DeRestPluginPrivate::addSensorNode(const deCONZ::Node *node, const SensorFi
             clusterId = VENDOR_CLUSTER_ID;
             sensorNode.addItem(DataTypeUInt16, RStateEventDuration);
         }
+        else if (modelId.startsWith(QLatin1String("ZBT-Remote-ALL-RGBW")))
+        {
+            sensorNode.addItem(DataTypeUInt16, RStateX);
+            sensorNode.addItem(DataTypeUInt16, RStateY);
+            sensorNode.addItem(DataTypeUInt16, RStateTiltAngle);
```

Comment:
Shouldn't this be RStateAngle instead of RStateTiltAngle? From my testing so far it shows "tiltangle":null in the REST API instead of the angle.

Reply:
Indeed, thanks for noticing!

It's correct in `database.cpp`.  If you restart deCONZ, you should see `state.angle` instead.